### PR TITLE
fix backend dl check

### PR DESCRIPTION
**Pull request by tim738745**

### CODE REVIEW

File: django/api/viewsets/application_form.py [hunk 1]
```diff
@@ -42,23 +42,22 @@ def get_serializer_class(self):
...
+        dl_not_valid = (
+            GoElectricRebateApplication.objects.filter(
+                drivers_licence__exact=drivers_licence
+            )
+            .filter(
+                status__in=[
+                    GoElectricRebateApplication.Status.SUBMITTED,
+                    GoElectricRebateApplication.Status.VERIFIED,
+                    GoElectricRebateApplication.Status.APPROVED,
+                    GoElectricRebateApplication.Status.REDEEMED,
```
Comment (naomiaro) [inline django/api/viewsets/application_form.py:55]: This should also have household initiated :)

**Review by naomiaro — CHANGES_REQUESTED** [review]
Hmm I think this would also have to check the issued rebates table if all historical data will be loaded there.

**Comment by naomiaro** [comment]
Maybe historic data can be loaded into the main application table instead though. We'll get the cleaned data tomorrow.

**Comment by tim738745** [comment]
> Maybe historic data can be loaded into the main application table instead though. We'll get the cleaned data tomorrow.

Ok, I'll leave it like this for now until we know what we want to do.

**Review by naomiaro — APPROVED** [review]

**Pull request merged**